packages/web3-core-helpers/src/formatters.js

```diff
@@ -239,7 +239,14 @@ var outputTransactionFormatter = function (tx) {
         tx.transactionIndex = utils.hexToNumber(tx.transactionIndex);
     tx.nonce = utils.hexToNumber(tx.nonce);
     tx.gas = utils.hexToNumber(tx.gas);
-    tx.gasPrice = outputBigNumberFormatter(tx.gasPrice);
+    if (tx.gasPrice !== null)
```

Comment:
```suggestion
    if (tx.gasPrice)
```

Reply:
do we even need this check?

Reply:
Nvm I think the original is correct the node will return bull on an empty value.

Reply:
Correct, that was the main reason I put together this PR, its coming back `null` in certain instances and throwing errors.